amethyst_renderer/src/lib.rs

```diff
@@ -209,7 +220,37 @@ impl Renderer {
     pub fn create_pipe(&mut self, pb: PipelineBuilder) -> Result<Pipeline> {
         pb.build(&mut self.factory, &self.main_target)
     }
-
+	/// Updates the rendering target (useful for resizes)
+	/// Works only on tux for now >_<
```

Comment:
What would need to be done to get this working elsewhere?

Reply:
I actually forgot about that comment, I'm installing the windows sdk to test that now.

Reply:
Fixed